docs/basics/functions.md

```diff
@@ -917,6 +918,97 @@ Which would return the something like following:
  }]
 ```
 
+**Example 2 (split_array and leaf_array)**
+
+Consider a service that returns the following payload:
+```json
+{
+    "concurrent_plays": {
+        "timestamps": [
+            1585662957000,
+            1585662958000,
+            1585662959000
+        ],
+        "meta": {
+            "status": 0
+        },
+        "type": "time_series",
+        "filters": {
+            "AccountA": [
+                200,
+                190,
+                180
+            ]
+        }
+    }
+}
+```
+
+You could split the leaf nodes within `timestamps` and `filters` branches into separate samples, then merge and join them together.
```

Comment:
s/then merge and join them together/which you could then merge and join.

Reply:
Updated functions.md accordingly.